### PR TITLE
Fixed annotations for GetOne()

### PR DESCRIPTION
**Pull request by HaaseIT**

### CODE REVIEW

File: adodb.inc.php [hunk 1]
```diff
@@ -1709,13 +1709,14 @@ function CacheGetAssoc($secs2cache, $sql=false, $inputarr=false,$force_array = f
...
-	* @param sql			SQL statement
-	* @param [inputarr]		input bind array
-	*/
-	function GetOne($sql,$inputarr=false) {
+	 * Return first element of first row of sql statement. Recordset is disposed
+	 * for you.
+	 *
+	 * @param string		$sql		SQL statement
+	 * @param array|bool	$inputarr	input bind array
+	 * @return string|int|bool
```
Comment (dregad) [inline adodb.inc.php]: I think it would be more appropriate to use a more generic `@return mixed`.

Comment (HaaseIT) [inline adodb.inc.php]: Whatever floats your boat :)

**Comment by dregad** [comment]
Thanks for your contribution

**Comment by HaaseIT** [comment]
Looking forward to use it in the next release :)

I will do more small fixes whenever they come up.

**Pull request merged**